addons/im_livechat/static/src/core/web/channel_invitation_patch.xml

```diff
@@ -1,9 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-inherit="discuss.ChannelInvitation.main" t-inherit-mode="extension">
+        <xpath expr="//*[@name='selectablePartnerDetail']//*" position="inside">
+            <span t-if="props.thread?.channel_type === 'livechat' and selectablePartner.user_livechat_username" class="overflow-hidden opacity-75 smaller mx-2">
+                <span class="text-truncate" t-esc="selectablePartner.user_livechat_username"/>
```

Comment:
Double `span` looks unnecessary, could move `text-truncate` to first one, in place of `overflow-hidden`.
This would actually fix missing ellipsis on second name.